### PR TITLE
Add osc exec and port-forward commands

### PR DESCRIPTION
**Pull request by ncdc**

### CODE REVIEW

**Comment by bparees** [comment]
lgtm.

**Comment by smarterclayton** [comment]
Add these to the end to end test.

**Comment by ncdc** [comment]
I probably won't get to these until tomorrow. Right now I'm fixing a bug in pull by digest and working on v2 registry integration.

File: pkg/cmd/cli/cmd/wrappers.go [hunk 1]
```diff
@@ -121,3 +121,41 @@ Examples:
...
+}
+
+func NewCmdPortForward(fullName string, f *clientcmd.Factory) *cobra.Command {
+	cmd := f.NewCmdPortForward()
+	longDesc := `Forward 1 or more local ports to a pod.
+
+Examples:
+
+	$ %[1]s port-forward -p mypod 5000 6000
+	<listens on ports 5000 and 6000 locally, forwarding data to/from ports 5000 and 6000 in the pod>
```
Comment (kargakis) [inline pkg/cmd/cli/cmd/wrappers.go]: If you don't mind, try keeping these examples consistent with the rest of the examples.

Comment (ncdc) [inline pkg/cmd/cli/cmd/wrappers.go]: It's copied from upstream. But I can change it if you want.

Comment (kargakis) [inline pkg/cmd/cli/cmd/wrappers.go]: Given that they are usage messages, accessible by a user I think we should take care of them;)

**Comment by ncdc** [comment]
[test]

File: pkg/cmd/cli/cmd/wrappers.go (hunk 1 above)
Comment (ncdc) [inline pkg/cmd/cli/cmd/wrappers.go]: @kargakis fixed, PTAL

Comment (kargakis) [inline pkg/cmd/cli/cmd/wrappers.go]: lgtm

**Comment by smarterclayton** [comment]
[merge]

**Comment by ncdc** [comment]
Merge failed because socat isn't installed in the ami. Will this fix it? https://github.com/openshift/vagrant-openshift/pull/225

**Comment by ncdc** [comment]
Base image now has socat, so let's try this again [merge]

**Pull request merged**